### ISSUE TITLE
Download old version in developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz

### ISSUE THREAD

**Issue by thvd** [forcedotcom/cli#1523]
### Summary

When you access https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz normally you would download the latest stable. But as of today, it's an old version: `sfdx-cli-v7.82.0-3d0c527ac6-linux-x64`

### Steps To Reproduce:

1. Download https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz
2. Unpack
3. See old version  `sfdx-cli-v7.82.0-3d0c527ac6-linux-x64`

### Expected result

Download the latest stable version.

### Actual result

Downloaded an old version: sfdx-cli-v7.82.0-3d0c527ac6-linux-x64

**Comment by daveespo** [comment]
@thvd  -- that's no longer the URL to download the tarball. The URLs for the tarballs are documented over here:

https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_linux

**Comment by iowillhoit** [comment]
I am going to go ahead and close this one, feel free to reopen if needed @thvd. Thanks @daveespo for helping out :)

**Comment by thvd** [comment]
@daveespo @iowillhoit  I see, but till yesterday this did just work fine, so something looks like changed? 
I have no problem with the switch, in fact i already migrated to the other. But it might be confusing to have it retrieve an ancient version?

**Comment by cristiand391** [comment]
Hey @thvd afaik, that url always pointed to v7.82, what I saw from other customers using it is that they were getting sfdx autoupdated so most of the time they didn't noticed the old version.

Google keeps showing docs like this when looking for install instructions: https://developer.salesforce.com/docs/atlas.en-us.222.0.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm

so a few people does use that URL, we can't update those docs but we might plan to actually remove that old tarball in the future.

Sorry for the confusion, we noticed it a few months ago but there's nothing we can do other than redirect people to the latest docs: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_linux

Hope that answers your question.
